### PR TITLE
Openx: Pass through imp.ext wholesale except prebid, bidder

### PR DESCRIPTION
**Pull request by laurb9**
Alternative to #12, pass all of `imp.ext` except `prebid` and `bidder`, merged with `customParameters` from `imp.ext.bidder`

### CODE REVIEW

File: adapters/openx/openx.go [hunk 1]
```diff
@@ -140,17 +137,22 @@ func preprocess(imp *openrtb2.Imp, reqExt *openxReqExt) error {
 		imp.BidFloor = openxExt.CustomFloor
 	}
 
+	// copy imp.ext to outgoing request imp.ext, except for prebid fields
```
Comment (kenan-gillet) [inline adapters/openx/openx.go]: commit message says "except prebid, bidder"

Comment (laurb9) [reply adapters/openx/openx.go]: I meant the prebid specific fields but I see how that didn't come out the way I intended

File: adapters/openx/openx.go [hunk 2]
```diff
@@ -140,17 +137,22 @@ func preprocess(imp *openrtb2.Imp, reqExt *openxReqExt) error {
...
+	delete(impExt, "bidder")
+	delete(impExt, "prebid")
+	addImpExt := len(impExt) > 0
 
 	if openxExt.CustomParams != nil {
-		impExt.CustomParams = openxExt.CustomParams
-		addImpExt = true
-	}
-	if bidderExt.AuctionEnvironment != openrtb_ext.ServerSideAuction {
-		impExt.AuctionEnvironment = bidderExt.AuctionEnvironment
```
Comment (kenan-gillet) [inline adapters/openx/openx.go:151]: don't we want to keep 
```impExt.AuctionEnvironment = bidderExt.AuctionEnvironment``` ?

Comment (laurb9) [reply adapters/openx/openx.go:151]: It is now copied as part of `imp.ext.ae`. The [fledge test](https://github.com/prebid/prebid-server/blob/master/adapters/openx/openxtest/exemplary/fledge-no-bids.json) checks for it, although I suppose it couldn't hurt to also add it here

**Review by kenan-gillet — APPROVED** [review]

**Review by kenan-gillet — APPROVED** [review]

**Pull request closed without merging**